core/thread/fetch_spec.rb

```diff
@@ -0,0 +1,34 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.5' do
+  describe 'Thread#fetch' do
+    it 'returns the value of the fiber-local varible if value has been assigned' do
+      th = Thread.new { Thread.current[:cat] = 'meow' }
+      th.join
+      th.fetch(:cat, true).should == 'meow'
+    end
+
+    it "returns the default value if fiber-local variable hasn't been assigned" do
+      th = Thread.new {}
+      th.join
+      th.fetch(:cat, true).should == true
+    end
+
+    it 'raises a KeyError when the Thread does not have a fiber-local variable of the same name' do
+      th = Thread.new {}
+      th.join
+      -> { th.fetch(:cat) }.should raise_error(KeyError)
+    end
+
+    it 'returns the value of the fiber-local varible if value has been assigned' do
```

Comment:
This spec has the same description as the first one.
This is problematic when tagging (excluding) a spec.
I think an elegant solution here would be to wrap sepcs 1-2 in `describe "with 2 arguments" do` and 3-4 in `describe "with 1 argument" do`

Reply:
And `varible` => `variable` :)

Reply:
done. I have updated the PR.